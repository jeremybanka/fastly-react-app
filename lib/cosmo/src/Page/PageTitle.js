/* @flow */

import * as React from "react";
import { Box } from "../";

type Props = { children: React.Node };

const PageTitle = (props: Props) => (
  <Box as="h1" margin="xxs" fontSize="xxl" fontWeight="bold">
    {props.children}
  </Box>
);

export default PageTitle;
