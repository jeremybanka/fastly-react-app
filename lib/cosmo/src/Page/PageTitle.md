The `<Page.Title />` component should be nested inside of a `<Page />`.
