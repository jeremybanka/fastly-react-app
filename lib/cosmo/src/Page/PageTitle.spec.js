/* @flow */

import React from "react";
import { shallow, render } from "enzyme";

import PageTitle from "./PageTitle";

describe("PageTitle test suite", () => {
  it("renders with className .PageTitle", () => {
    expect(shallow(<PageTitle>Page Title</PageTitle>).is(".PageTitle")).toBe(
      true
    );
  });

  it("renders the stored snapshot", () => {
    expect(render(<PageTitle>Page Title</PageTitle>)).toMatchSnapshot();
  });
});
