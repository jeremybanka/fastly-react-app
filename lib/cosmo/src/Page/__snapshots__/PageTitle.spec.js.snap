// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle test suite renders the stored snapshot 1`] = `
<h1
  class="PageTitle"
>
  Page Title
</h1>
`;
